MogoPocTest/CountryControllerUnitTest.cs

```diff
@@ -0,0 +1,34 @@
+using Bks.DataAccess.Mongo;
+using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MongoPocWebApplication1.ControllersPresentationAndApplication;
+using MongoPocWebApplication1.Domain.Entities;
+using MongoPocWebApplication1.Domain.Repositories;
+
+namespace MogoPocTest
+{
+    [TestClass]
+    public class CountryControllerUnitTest
+    {
+        private readonly ICountryRepository countryRepositoryMock = A.Fake<ICountryRepository>();
+        private readonly ILogger<MongoConnector> mockLogger = A.Fake<ILogger<MongoConnector>>();
+        
+        CountryControllerUnitTest()
+        {
+
+        }
+        [TestMethod]
+        public async void Post_Country_ShouldInvokeAddAsyncOnRepo()
```

Comment:
The test is redundand

Reply:
This was just a template sample for a integration test, showing how we can test the Business Logic with a mocked Repository implementation.